templates/fine-tune-llm_v2/cookbooks/run_llmforge_as_anyscale_jobs/README.ipynb

```diff
@@ -0,0 +1,104 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "\n",
+    "# Running LLM finetuning template as an Anyscale job\n",
+    "**⏱️ Time to complete**: 10 minutes\n",
+    "\n",
+    "While for developer velocity it is preferable to use Workspace to run python scripts, for automation and launching jobs from your laptop without having to spin up a Workspace it is recommended to run the fine-tuning workloads via isolated [Anyscale jobs](https://docs.anyscale.com/platform/jobs/).\n",
+    "\n",
+    "We have to specify the command that needs to run (ex. [COMMAND][ARGS]) along with the requirements (ex. docker image, additional, pip packages, etc.) in a job yaml and then call `anyscale job submit` to launch the job on Anyscale.\n",
+    "\n",
+    "> Note: Executing an Anyscale Job within a Workspace will ensure that files in the current working directory are available for the Job (unless excluded with --exclude). But we can also load files from anywhere (ex. Github repo, S3, etc.) if we want to launch a Job from anywhere."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "name: \"llmforge-job\"\n",
+      "entrypoint: \"llmforge anyscale finetune config/llama-3-8b.yaml\"\n",
+      "image_uri: localhost:5555/anyscale/llm-forge:0.5.0.1-ngmM6BdcEdhWo0nvedP7janPLKS9Cdz2\n",
+      "requirements: []\n",
+      "max_retries: 0\n",
+      "working_dir: \".\""
+     ]
+    }
+   ],
+   "source": [
+    "!cat ./job_configs/job.yaml"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "A few notes:\n",
+    "\n",
+    "- `entrypoint` is basically the command we want to run. Pay attention to the relative file location (config/llama-3-8b.yaml) and the `working_dir`. Inside `llama-3-8b.yaml` we are also referencing a relative path to `config/zero_3_offload_optim+param.json` which works because of specifying the `working_dir` to be the current dir `.`.\n",
```

Comment:
```suggestion
    "- `entrypoint` is basically the command we want to run. Pay attention to the relative file location (config/llama-3-8b.yaml) and the `working_dir`. Inside `llama-3-8b.yaml` we are also referencing a relative path to `config/zero_3_offload_optim+param.json`. This works because we specify `working_dir` to be the current directory `.`.\n",
```